app/services/accept_offer.rb

```diff
@@ -6,9 +6,10 @@ def initialize(application_choice:)
   def save!
     ActiveRecord::Base.transaction do
       ApplicationStateChange.new(@application_choice).accept!
+      @application_choice.update(accepted_at: Time.zone.now)
 
       other_application_choices_with_offers.each do |other_application_choice|
-        ApplicationStateChange.new(other_application_choice).decline!
+        DeclineOffer.new(application_choice: other_application_choice).save!
```

Comment:
Much better! This, however, may trigger multiple Slack notifications for the automatic declines. Are we ok with that?

Reply:
I decided we were, but open to being convinced the other way. We could always flag it if it became annoying

Reply:
Oh wait we have the block which silences Slack... let me find it